artifactory/commands/generic/search.go

```diff
@@ -122,3 +158,36 @@ func GetSearchParams(f *spec.File) (searchParams services.SearchParams, err erro
 	searchParams.IncludeDirs, err = f.IsIncludeDirs(false)
 	return
 }
+
+func PrintSearchResults(reader *content.ContentReader) error {
```

Comment:
The logic of this function can probably be simplified so that it is easier to read and understand.
Let's also add a unit test for this function.